cluster/charts/library/templates/_recommended-labels.tpl

```diff
@@ -0,0 +1,9 @@
+{{/*
+Common labels
+*/}}
+{{- define "library.rook-ceph.labels" -}}
+app.kubernetes.io/part-of: rook-ceph
```

Comment:
I'm looking at this and starting to wonder if we should make `part-of` match more between our operator-created resources and helm-created resources. I'm also wondering if we should bother with this for RBAC resources. Maybe we should leave this out of the recommended-labels template and instead just apply labels to the rook-ceph operator deployment.

Reply:
Do you mean keeping only the 2 commit changes and removing the first commit changes?

Reply:
What if we set `part-of: rook-operator` wherever it's hard-coded, and the parent CR name wherever the operator generates the resource?